lib/hawkular/hawkular_client.rb

```diff
@@ -66,7 +66,7 @@ def tokens
 
     # this is in a dedicated method, because constructor opens the websocket connection to make the handshake
     def init_operations_client
-      Operations::OperationsClient.new(entrypoint: @state[:entrypoint],
+      Operations::Client.new(entrypoint: @state[:entrypoint],
```

Comment:
I don't think so as long as we're using it only internally (and we can deprecate using `#new` directly), the user is not expected to be running `Operations::Client` directly, is it?